cf-key/cf-key.c

```diff
@@ -354,6 +354,12 @@ static GenericAgentConfig *CheckOpts(int argc, char **argv)
             }
 
         case 'C':
+            /* hack to get optional argument with short option */
```

Comment:
This comment definitely deserves a bit more information about what's needed and why exactly.